qiskit/transpiler/passes/scheduling/alap.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, *args, **kwargs):
             "which performs the as analysis pass that requires a padding pass to later modify "
             "the circuit. This class will be deprecated in a future release and subsequently "
             "removed after that.",
-            PendingDeprecationWarning,
+            DeprecationWarning,
```

Comment:
Here (and probably in others) the message says "will be deprecated", but we're updating the warning so it now _is_ deprecated.

Reply:
Heh, yeah that should have been updated. That's what I get for just find replacing the warning name. Fixed in: https://github.com/Qiskit/qiskit-terra/pull/8023/commits/f8a92dc5aa7105ce426c1bcd28d35203ef7ce0e2